app/helpers/application_helper.rb

```diff
@@ -1176,7 +1177,8 @@ def render_listnav_filename
   def show_adv_search?
     show_search = %w(availability_zone cim_base_storage_extent cloud_volume cloud_volume_snapshot container_group container_node container_service
                      container_route container_project container_replicator container_image container_image_registry
-                     persistent_volume container_build
+                     persistent_volume container_build ems_middleware middleware_server middleware_domain
+                     middleware_messaging middleware_deployment middleware_datasource
```

Comment:
Same as above, groups are not visible from navigation so I did not include them in search.